File: lib/py/uapi/internal/binary/UnpackBody.py
from typing import object, dict


def unpack_body(body: dict[object, object]) -> dict[object, object]:
    from uapi.internal.binary.Unpack import unpack

    result = {}

    for key, value in body.items():
        unpacked_value = unpack(value)
        result[key] = unpacked_value

    return result

File: lib/ts/uapi/internal/binary/UnpackBody.ts
import { unpack } from './Unpack';

export function unpackBody(body: Map<any, any>): Map<any, any> {
    const result: Map<any, any> = new Map();

    for (const [key, value] of body.entries()) {
        const unpackedValue = unpack(value);
        result.set(key, unpackedValue);
    }

    return result;
}
